requirements.txt

```diff
@@ -1,2 +1,3 @@
+mdl=0.4.0
```

Comment:
mdl is Ruby-based, requirements.txt is for Python libs. This PR won't do what you think it will do :)

Reply:
Ouch...